### DISCUSSION TITLE
Add support for Tree and List/Table widgets

### DISCUSSION THREAD

**Discussion by alexmocanu** [AparokshaUI/Adwaita#40] [Improvements]
Hello,

I've been trying to build a grid/table view to display some data, but none of the existing widgets seem to fit. 
Could it be possible to add support for these please?

https://docs.gtk.org/gtk4/section-tree-widget.html

**Comment by david-swift** [comment]
Hello!

There currently is support for simple grid views with the [FlowBox](https://aparokshaui.github.io/adwaita-swift/documentation/adwaita/flowbox) widget. Does this work in your case?

I can add support for the GtkTreeView either this weekend or at some point after next week.
